### PR TITLE
[INLONG-396][Release] Add blog for the 1.2.0 release

### LINKED ISSUE #396: [Release] Add blog for the 1.2.0 release
### Description

Add blog for the 1.2.0 release.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
Fixes #396 

### Motivation

Add blog for the 1.2.0 release.

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-blog/2022-06-22-release-1.2.0.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+更多特性信息，请参考：[InLong Sort ETL 方案解析](2022-06-16-inlong-sort-etl_ch.md)
+
+### 其他特性及问题修复
+相关内容请参考 [版本说明](https://github.com/apache/inlong/blob/master/CHANGES.md) ，其中详细列出了此版本的特性、提升和 Bug 修复。
+
+## Apache InLong 后续规划
+后续版本，我们扩展更多的数据源端和存储端，覆盖更多的使用场景，并逐步提升系统的易用性和健壮性，主要包括：
+- 各组件的心跳上报
+- 数据链路的状态管理
+- 补充系统的审计和监控能力
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-blog/2022-06-22-release-1.2.0.md:94]: ->
全链路审计支持写入 ClickHouse

Comment (healchow) [reply i18n/zh-CN/docusaurus-plugin-content-blog/2022-06-22-release-1.2.0.md:94]: OK.

File: blog/2022-06-16-inlong-sort-etl.md [hunk 2]
```diff
@@ -19,9 +18,9 @@ First of all, based on Apache Flink SQL, there are mainly the following consider
 
 **Note**:  for all codes of this scheme, please refer to [Apache inlong sort]（ https://github.com/apache/incubator-inlong/tree/master/inlong-sort ）Module, which can be downloaded and used in the upcoming version 1.2.0.
```
Comment (yunqingmoswu) [inline blog/2022-06-16-inlong-sort-etl.md:19]: for all codes of this scheme -> For all codes of this scheme

Comment (healchow) [reply blog/2022-06-16-inlong-sort-etl.md]: done.

**Review by yunqingmoswu — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Review by leosanqing — APPROVED** [review]

**Review by leezng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**